src/rofl/common/openflow/experimental/matches/capwap_matches.h

```diff
@@ -18,22 +18,24 @@ enum oxm_ofx_match_fields {
   OFPXMT_OFX_CAPWAP_FLAGS = 29,
 };
 
+static unsigned int const OXM_EXP_ID_SIZE = sizeof(uint32_t); // length of experimenter id field
```

Comment:
would have been nice if this was only in a single file and not redefined in all of the experimental